samples/chat/README.md

```diff
@@ -88,7 +86,29 @@ Please refer to the root level [README](../../README.md#requirements) for prereq
     Who won the SuperBowl in 2014?
     ```
 
-    The response should be an HTTP 202 Accepted response with an empty body. This means the chatbot is processing the request.
+    The response should look something like the following exampl with Status 200 OK, formatted for readability.
```

Comment:
Per discussion, the HTTP 200 response for the HTTP POST should contain only the immediate response and not the full history.